### PR TITLE
Create Introduction to Expensify Support

### PR DESCRIPTION
**Pull request by isabelastisser**
Adding the Expensify Support resource to the help site.

Original[ GH](https://github.com/Expensify/Expensify/issues/311414)

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/311414
PROPOSAL: 

### Tests

- [x] Verify that no errors appear in the JS console

### QA Steps

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [ ] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] If we are not using the full Onyx data that we loaded, I've added the proper selector in order to ensure the component only re-renders when the data it is using changes
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

</details>

<details>
<summary>Android: mWeb Chrome</summary>

</details>

<details>
<summary>iOS: Native</summary>

</details>

<details>
<summary>iOS: mWeb Safari</summary>

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

</details>

<details>
<summary>MacOS: Desktop</summary>

</details>

### CODE REVIEW

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support
Comment (MonilBhavsar) [inline docs/articles/new-expensify/getting-started/Introduction to Expensify Support:1]: We need title and description, right?
```MD
---
title: 
description: 
---
<!-- The lines above are required by Jekyll to process the .md file -->
```

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support [hunk 1]
```diff
@@ -0,0 +1,175 @@
+Expensify offers comprehensive support for customers of all sizes and accommodates various customer needs, from entrepreneurs to multinational corporations.
+
+**Concierge Support**
+
+- Concierge is available 24/7 and is the first line of support.
+- Users can access Concierge through the Expensify app or via emailconcierge@expensify.com.
```
Comment (MonilBhavsar) [inline docs/articles/new-expensify/getting-started/Introduction to Expensify Support:6]: Right?
```suggestion
- Users can access Concierge through the Expensify app or via emailing to [concierge@expensify.com](mailto:concierge@expensify.com).
```

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+- Don’t be your team’s tech support - refer your users to Concierge for all their Expensify product questions.
+
+**Setup Specialists**
+
+- Dedicated onboarding experts to assist users in getting started with Expensify.
+- Assist in connecting cards and bank accounts, configuring integrations, and more.
+
+**Account Managers**
+
+- A dedicated point of contact for companies with more than ten users
```
Comment (MonilBhavsar) [inline docs/articles/new-expensify/getting-started/Introduction to Expensify Support:17]:
```suggestion
- A dedicated point of contact for companies with more than ten users.
```

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+- Offer an additional support layer for accounting partners.
+- Partner Managers are dedicated Expensify team members.
+- Provide firmwide training, assist with strategic goal planning, collaborative marketing, and more.
+
+This comprehensive support structure ensures that Expensify users receive the assistance they need, whether new to the platform or looking to optimize their experience.
+
+# How to get support in Expensify
+
+If you have any questions or need assistance, please email us at concierge@expensify.com. If you're using our mobile app, you can also message us through the chat bubble in the top right corner. Alternatively, click your user icon on the web app to find the 'Help and Feedback' option. We're here to help!
```
Comment (MonilBhavsar) [inline docs/articles/new-expensify/getting-started/Introduction to Expensify Support:37]:
```suggestion
If you have any questions or need assistance, please email us at [concierge@expensify.com](mailto:concierge@expensify.com). If you're using our mobile app, you can also message us through the chat bubble in the top right corner. Alternatively, click your user icon on the web app to find the 'Help and Feedback' option. We're here to help!
```

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+
+## Concierge
+
+Concierge is your personal assistant for managing expenses and is accessible through chat, ready to assist you whenever you need support! It's here to help you create expenses, submit finished reports, and even assist with approvals. It's like having a virtual helper who can also handle employee reimbursements. The more you use Concierge, the better it becomes at making your expense process smoother.
+
+## Your Setup Specialist
+
+Expensify provides phone support for anyone through a Setup Specialist! Your dedicated Setup Specialist can be reached at new.expensify.com using our powerful chat tool. 
+
+#What to Expect from your Setup Specialist:
```
Comment (MonilBhavsar) [inline docs/articles/new-expensify/getting-started/Introduction to Expensify Support:47]:
```suggestion
# What to Expect from your Setup Specialist:
```

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+
+## Account Managers
+
+If you have more than ten active users, your Account Manager will pick up where your Setup Specialist left off at the end of your official onboarding. Your account manager is a dedicated point of contact to support Workspace admins with questions about their Expensify account. They are available to help you and other Workspace admins review your account, advise on best practices, and make changes to your Workspace on your behalf whenever you need a hand. They will actively monitor open technical issues and be proactive with recommendations to increase efficiency and minimize time spent on expense management.
+Unlike Concierge, an account manager’s support will not be real-time, 24 hours a day. Your account manager will be super responsive when online, but anything sent when they’re offline will not be responded to until they’re online again.
+
+**For real-time responses and simple troubleshooting issues, you can always message our general support by writing to Concierge via the in-product chat or by emailing concierge@expensify.com.**
```
Comment (MonilBhavsar) [inline docs/articles/new-expensify/getting-started/Introduction to Expensify Support:66]:
```suggestion
**For real-time responses and simple troubleshooting issues, you can always message our general support by writing to Concierge via the in-product chat or by emailing [concierge@expensify.com](mailto:concierge@expensify.com).**
```

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+
+We make it easy to contact your account manager:
+
+1. Log in to your Expensify account, click “Support” along the left side of the page, and click the “Account Manager” option
+2. Reply to (or click the chat link on) any email you get from your account manager
+3. Sign in to new.expensify.com and go to the #admins room to access the chats for your Workspaces. Your account manager is in your #admin rooms, ready to help you, so you can ask for help there, and your account manager will respond in the chat.
+
+## Partner Managers
+
+A Partner Manager is a dedicated point of contact to support our ExpensifyApproved! Accountants with questions about their Expensify account. Partner Managers support our accounting partners by providing recommendations for client’s accounts, assisting with firm-wide training, and ensuring partners receive the full benefits of our partnership program. They will actively monitor open technical issues and be proactive with recommendations to increase efficiency and minimize time spent on expense management.
```
Comment (MonilBhavsar) [inline docs/articles/new-expensify/getting-started/Introduction to Expensify Support:81]:
```suggestion
A Partner Manager is a dedicated point of contact to support our ExpensifyApproved! Accountants with questions about their Expensify account. Partner Managers support our accounting partners by providing recommendations for client’s accounts, assisting with firmwide training, and ensuring partners receive the full benefits of our partnership program. They will actively monitor open technical issues and be proactive with recommendations to increase efficiency and minimize time spent on expense management.
```

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+
+## Getting Answers from Concierge
+
+For any Expensify basics, just ask Concierge. Try questions like:
+
+- What is SmartScan?
+- How do I add a mileage expense from the mobile app?
+- Where can I add a Secondary Login?
+
+Tips When contacting Expensify Support:
```
Comment (MonilBhavsar) [inline docs/articles/new-expensify/getting-started/Introduction to Expensify Support:109]:
```suggestion
Tips when contacting Expensify Support:
```

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+
+This feature is specifically for our new customers! Whenever you start a free trial, a product setup specialist will be assigned to guide you through the setup process.
+
+## What are the #announce and #admins rooms at new.expensify.com?
+
+When you create a Workspace on the Free Plan, you'll automatically gain access to two chat rooms:
+
+**#announce**: This room is for important announcements and is the default open to all Workspace members.
+
+**#admins** - every Workspace Admin will be invited to this room by default.
```
Comment (MonilBhavsar) [inline docs/articles/new-expensify/getting-started/Introduction to Expensify Support:147]:
```suggestion
**#admins** - every Workspace admin will be invited to this room by default.
```

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+Unlike direct messages, rooms have three notification settings. Members can adjust this room-by-room by clicking the chat header and accessing the Details page on the right-hand side.
+
+- **Daily** (enabled by default) - A daily digest of all email/SMS messages is sent at 9 am in the member’s local timezone, summarising all the unread messages in the room that were missed in the previous 24-hour window.
+- **Always** - Push notifications are sent in real time, adding an email/SMS fallback for any chats missed in the previous 10 minutes.
+- **Mute** - Push notifications and email/SMS fallback notifications are not sent at all.
+
+## How do I know if my Partner Manager is online?
+
+You can see if they are online via their status in new.expensify.com, which will either say “online” or have their working hours.
+## What if I’m unable to reach my Partner Manager?
```
Comment (MonilBhavsar) [inline docs/articles/new-expensify/getting-started/Introduction to Expensify Support:160]:
```suggestion

## What if I’m unable to reach my Partner Manager?
```

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support [hunk 10]
```diff
@@ -0,0 +1,175 @@
...
+If you cannot contact your Partner Manager (i.e., they’re out of the office for the day), you can contact Concierge for assistance. Your Partner Manager will get back to you when they’re online again.
```
Comment (MonilBhavsar) [inline docs/articles/new-expensify/getting-started/Introduction to Expensify Support:161]:
```suggestion

If you cannot contact your Partner Manager (i.e., they’re out of the office for the day), you can contact Concierge for assistance. Your Partner Manager will get back to you when they’re online again.

```

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support [hunk 11]
```diff
@@ -0,0 +1,175 @@
...
+## Can I get on a call with my Partner Manager?
+Of course! You can ask your Partner Manager to schedule a call whenever you think one might be helpful. Partner Managers can discuss client onboarding strategies, comprehensive firm training, and client setups.
```
Comment (MonilBhavsar) [inline docs/articles/new-expensify/getting-started/Introduction to Expensify Support:163]:
```suggestion

Of course! You can ask your Partner Manager to schedule a call whenever you think one might be helpful. Partner Managers can discuss client onboarding strategies, comprehensive firm training, and client setups.

```

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support [hunk 12]
```diff
@@ -0,0 +1,175 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (MonilBhavsar) [inline docs/articles/new-expensify/getting-started/Introduction to Expensify Support:175]: We need to remove all these lines at the end. I cannot make a suggestion
```suggestion
```

**Review by MonilBhavsar — COMMENTED** [review]
Could you also please link issue for reference

**Comment by isabelastisser** [comment]
@MonilBhavsar 

> Could you also please link issue for reference

There you go: https://github.com/Expensify/Expensify/issues/311414

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support (hunk 1 above)
Comment (isabelastisser) [reply docs/articles/new-expensify/getting-started/Introduction to Expensify Support:6]: Small change:

- Users can access Concierge through the Expensify app or via email by writing to [concierge@expensify.com](mailto:concierge@expensify.com).

**Review by MonilBhavsar — COMMENTED** [review]
Thanks!
Tests are failing because of route. Looks like we need to rename file to `Introduction-to-Expensify-Support.md` as per our naming convention
And add metadata https://github.com/Expensify/App/pull/30531/files#r1375908987

@isabelastisser @rushatgabhane

File: docs/articles/new-expensify/getting-started/Introduction to Expensify Support (hunk 1 above)
Comment (MonilBhavsar) [reply docs/articles/new-expensify/getting-started/Introduction to Expensify Support:6]: Updated suggestion

**Comment by isabelastisser** [comment]
Hey @MonilBhavsar, can you please clarify? Do I need to action this? Thanks!

**Comment by MonilBhavsar** [comment]
Yes, Either of you can action this. We need to rename the file and add a metadata. Happy to help.

**Comment by isabelastisser** [comment]
@MonilBhavsar, I have no idea how to rename the file and add a metadata. Do we have instructions in SO? Can you please explain? @rushatgabhane Thanks!

**Comment by maddylewis** [comment]
adding this resource via this PR instead - https://github.com/Expensify/App/pull/31319

**Pull request closed without merging**